src/m365/spo/commands/site/site-set.ts

```diff
@@ -270,61 +400,93 @@ class SpoSiteSetCommand extends SpoCommand {
       });
   }
 
-  private updateSharedProperties(logger: Logger, args: CommandArgs): Promise<void> {
-    if (typeof args.options.classification === 'undefined' &&
-      typeof args.options.disableFlows === 'undefined' &&
-      typeof args.options.shareByEmailEnabled === 'undefined') {
-      return Promise.resolve();
+  private updateSiteProperties(logger: Logger, args: CommandArgs): Promise<string> {
+    const isGroupConnectedSite = this.groupId !== '00000000-0000-0000-0000-000000000000';
```

Comment:
as this kind of condition is used twice maybe we could create a helper method for it 🤔?